venues/ICML.cc/2017/MLAV/webfield/conf.html

```diff
@@ -0,0 +1,97 @@
+
+// ------------------------------------
+// Basic venue homepage template
+//
+// This webfield displays the conference header (#header), the submit button (#invitation),
+// and a list of all submitted papers (#notes).
+// ------------------------------------
+
+// Constants
+var CONFERENCE = 'ICML.cc/2017/MLAV';
+var INVITATION = 'ICML.cc/2017/MLAV/-/Submission';
+var SUBJECT_AREAS = [
+  // Add conference specific subject areas here
+];
+var BUFFER = 1000 * 60 * 30;  // 30 minutes
+var PAGE_SIZE = 50;
+
+var paperDisplayOptions = {
+  pdfLink: true,
+  replyCount: true,
+  showContents: true
+};
+
+// Main is the entry point to the webfield code and runs everything
+function main() {
+  Webfield.ui.setup('#group-container', CONFERENCE);  // required
+
+  renderConferenceHeader();
+
+  load().then(render).then(function() {
+    Webfield.setupAutoLoading(INVITATION, PAGE_SIZE, paperDisplayOptions);
+  });
+}
+
+// RenderConferenceHeader renders the static info at the top of the page. Since that content
+// never changes, put it in its own function
+function renderConferenceHeader() {
+  Webfield.ui.venueHeader({
+    title: 'ICML 2017 MLAV Workshop',
+    subtitle: 'Machine Learning for Autonomous Vehicles',
+    location: 'International Conference on Machine Learning, Sydney, Australia',
+    date: '2017-08-10 17:00:00',
+    website: 'https://sites.google.com/site/ml4autovehicles2017/call-for-papers',
+    instructions: null,  // Add any custom instructions here. Accepts HTML
+    deadline: '2017-06-06 17:00:00'
```

Comment:
Ohh wait, may be the date is ok, and we only need to remove the deadline field, right?